Sources/StreamVideoSwiftUI/Livestreaming/LivestreamPlayerHelper.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright © 2025 Stream.io Inc. All rights reserved.
+//
+
+import StreamVideo
+import SwiftUI
+
+@MainActor
+class LivestreamPlayerHelper {
+    
+    private let formatter: DateComponentsFormatter = {
+        let formatter = DateComponentsFormatter()
+        formatter.unitsStyle = .positional
+        return formatter
+    }()
+    
+    func formatTimeInterval(_ interval: TimeInterval) -> String {
+        let totalSeconds = Int(interval)
+        let hours = totalSeconds / 3600
+        let minutes = (totalSeconds % 3600) / 60
+        let seconds = totalSeconds % 60
+
+        if hours > 0 {
+            return String(format: "%d:%02d:%02d", hours, minutes, seconds)
+        } else {
+            return String(format: "%d:%02d", minutes, seconds)
+        }
+    }
+    
+    func duration(from state: CallState) -> String? {
```

Comment:
Why not using the SDK provided `@Injected(\.formatters.mediaDuration) private var formatter: MediaDurationFormatter` and completely remove the `LivestreamPlayerHelper`?